### PR TITLE
s-zcracks: Use in ng mode

### PR DESCRIPTION
**Pull request by estreby**
Fix native useless detection for debian 9

### CODE REVIEW

File: scripts.d/s-salome-ng [hunk 1]
```diff
@@ -63,6 +64,8 @@ function setup-salome-ng-env()
             local VSET="no"
 
             if [[ $ACTION == SET ]]; then
+                echo $VAR >> /home/es4c769n/dev/build-salome-ng-dev/s-salome-ng/var.txt
```
Comment (TurboGit) [inline scripts.d/s-salome-ng:67]: some debug to be removed

File: scripts.d/.plan-ng-dev [hunk 2]
```diff
@@ -40,13 +41,20 @@ s-cmake/native
...
         s-persalys:skip
+        s-zcracks:skip
 ]
 
 [ = deb,9
         s-salome-ng:s9-dev
         s-salome-tar:#master:DB09
         s-cmake:skip
         s-zeromq:skip
+        s-cgns:skip
```
Comment (TurboGit) [inline scripts.d/.plan-ng-dev:52]: All those seems not related to zcracks module?

**Review by TurboGit — CHANGES_REQUESTED** [review]
Some changes are needed!

File: scripts.d/.plan-ng-dev (hunk 2 above)
Comment (TurboGit) [reply scripts.d/.plan-ng-dev:52]: Thanks for removing the debug, but what about all those skipped plug-ins? This can't be related to s-zcracks as it does not bring a single dependency, right? So to me all this is superfluous. Please clarify. TIA.

Comment (estreby) [reply scripts.d/.plan-ng-dev:52]: Yes you're right It doesn't concern Zcracks. I can remove it to push in another branch.

**Review by TurboGit — APPROVED** [review]
Looks good to me. I won't merge this over the week-end to avoid any possible issues with the build. So let's do the on Monday now. Thanks.
